src/toolchain-conventions.adoc

```diff
@@ -406,6 +406,29 @@ NOTE: This option does not affect inline assembly.
 The precedence among `-m[no]-scalar-strict-align`, `-m[no-]vector-strict-align`,
 and `-m[no-]strict-align` is determined by the last one specified.
 
+=== `-fcf-protection=[full|branch|return|none]`/`-fcf-protection`
+
+
+Enable control flow protection. The compiler will insert control flow integrity
+instructions to protect the program against control flow hijacking attacks.
+
+`-fcf-protection` is alias to `-fcf-protection=full`.
+
+- `none`: Disable control flow protection.
+- `full`: Protect all control flow instructions, will enable branch protection
```

Comment:
This sentence reads like branch protection requires Zimop, but I think only return protection requires Zimop?